sdk/eventhub/Azure.Messaging.EventHubs/src/Metadata/PartitionProperties.cs

```diff
@@ -9,7 +9,7 @@ namespace Azure.Messaging.EventHubs.Metadata
     ///   A set of information for a single partition of an Event Hub.
     /// </summary>
     ///
-    public sealed class PartitionProperties
+    public class PartitionProperties
```

Comment:
Somewhat orthogonal, I opened this up to allow library consumers to be able to mock when testing.